source/topics/troubleshooting/what-is-a-csr.md

```diff
@@ -0,0 +1,19 @@
+A CSR, or [Certificate Signing Request](https://en.wikipedia.org/wiki/Certificate_signing_request), is a message you generate with information about the SSL/TLS certificate you want your SSL/TLS certificate provider to issue. You encrypt the CSR with your private key.
+
+Using OpenSSL, [the req command](https://www.openssl.org/docs/manmaster/apps/req.html) can generate a new private key and CSR:
+
+```
+openssl req -newkey rsa:2048 -nodes -keyout [YOUR_DOMAIN].key -out [YOUR_DOMAIN].csr
+```
+
+Store the private key and CSR in a secure location. If your certificate provider asks, request an "NGiNX/other" format.
```

Comment:
This was a little unclear to me — when is the provider going to ask? I think you might be missing a sentence about uploading the CSR to a CA?

(you might also want to refer to them as a "certification authority").